### PR TITLE
Ex test

### PR DESCRIPTION
**Pull request by jbjharvey**
All done with Trial Homework 1....Finally!

### CODE REVIEW

File: exercises/trial_homework.rmd [hunk 1]
```diff
@@ -81,8 +79,8 @@ gnames <- c("cpetrik", "wildflowermt", "mad4mocha", "sjohnson216", "okisutch99",
 ords <- sample(length(gnames))
 
 # return the first 4 github names that were signed up
-submit_answer({
-
+submit_answer({names(gnames) <- c(ords)
```
Comment (eriqande) [inline exercises/trial_homework.rmd:82]: You don't need to assign `ords` to the `names` of gnames.

It is sufficient to just do `gnames[ords<=4]`

But your solution still delivers the correct result!

Also, no need to wrap `ords` in the `c()` function.

**Comment by eriqande** [comment]
Awesome!

Julio! You are da man!  Way to get these done.  They look great!.  I wrote a few comments on one line which you should be able to read above.

Super cool!

Catch you next week.

eric

**Comment by jbjharvey** [comment]
Thanks Eric!

I really appreciate your reply and corrections, very helpful!

Also, currently looking thorough "trial-homework-thoughts", noticed those
same corrections and some other excellent hints — thanks for your
thoroughness and attention to detail, so helpful!

Wouldn't you know it, I have to go up to Bodega marine lab to get samples
and meet with colleagues on Tuesday, will try to make it back for class but
we'll see.

Thanks again for all your help and understanding,

Julio

—
Julio Harvey, Ph.D.
Monterey Bay Aquarium Research Institute
7700 Sandholdt Road
Moss Landing, CA  95039
tel: 831.775.1789
fax: 831.775.1620
web: http://www.mbari.org/staff/jharvey/
—

On Thu, Nov 13, 2014 at 11:03 AM, Eric C. Anderson <notifications@github.com

**Comment by eriqande** [comment]
No worries if you don't make it back in time tuesday.  Drive safely and
have a good time in Bodega!

Best,

eric

**Comment by jbjharvey** [comment]
Many thanks Eric, I appreciate it!

Julio

—
Julio Harvey, Ph.D.
Monterey Bay Aquarium Research Institute
7700 Sandholdt Road
Moss Landing, CA  95039
tel: 831.775.1789
fax: 831.775.1620
web: http://www.mbari.org/staff/jharvey/
—

On Thu, Nov 13, 2014 at 11:59 AM, Eric C. Anderson <notifications@github.com

**Pull request closed without merging**